src/Stryker.Core/Stryker.Core/TestRunners/DotnetTestRunner.cs

```diff
@@ -1,20 +1,23 @@
 ﻿using Stryker.Core.Testing;
 using System.Collections.Generic;
+using Stryker.Core.Parsers;
 
 namespace Stryker.Core.TestRunners
 {
     public class DotnetTestRunner : ITestRunner
     {
+        private readonly ITotalNumberOfTestsParser _totalNumberOfTestsParser;
         private string _path { get; set; }
         private IProcessExecutor _processExecutor { get; set; }
 
-        public DotnetTestRunner(string path, IProcessExecutor processProxy)
+        public DotnetTestRunner(string path, IProcessExecutor processProxy, ITotalNumberOfTestsParser totalNumberOfTestsParser)
         {
+            _totalNumberOfTestsParser = totalNumberOfTestsParser;
             _path = path;
             _processExecutor = processProxy;
         }
 
-        public DotnetTestRunner(string path) : this(path, new ProcessExecutor()) { }
+        public DotnetTestRunner(string path) : this(path, new ProcessExecutor(), new TotalNumberOfTestsParser()) { }
```

Comment:
What is this empty constructor used for?

Reply:
It was removed.